bin/zkServer.sh

```diff
@@ -270,8 +270,8 @@ status)
       secureClientPort=`$GREP "^[[:space:]]*secureClientPort[^[:alpha:]]" "$ZOOCFG" | sed -e 's/.*=//'`
       if [ "$secureClientPort" ] ; then
         isSSL="true"
+        clientPort=$secureClientPort
       fi
-      clientPort=$secureClientPort
     fi
     if [ ! "$clientPort" ] ; then
```

Comment:
Can we refactor it like this?
```
if [ ! "$clientPort" ] ; then
      echo "Client port not found. Looking for secureClientPort in the static config."
      secureClientPort=`$GREP "^[[:space:]]*secureClientPort[^[:alpha:]]" "$ZOOCFG" | sed -e 's/.*=//'`
      if [ "$secureClientPort" ] ; then
        isSSL="true"
        clientPort=$secureClientPort
      else
        echo "Unable to find either secure or unsecure client port in any configs. Terminating."
        exit 1
    fi
fi
```

Reply:
yes, thanks!
you are right, it is a bit more compact this way. I changed.